zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/tenant/PersistedTenant.java

```diff
@@ -94,9 +100,7 @@ public PersistedTenant setName(final String name) {
    *
    * @param tenantRecord the TenantRecord from which to copy the data
    */
-  public void wrap(final TenantRecord tenantRecord) {
-    tenantKeyProp.setValue(tenantRecord.getTenantKey());
-    tenantIdProp.setValue(tenantRecord.getTenantId());
-    nameProp.setValue(tenantRecord.getName());
+  public void copy(final TenantRecord tenantRecord) {
+    copyFrom(tenantRecord);
```

Comment:
:x: This looks wrong to me, you are copying a `TenantRecord` into a `PersistedTenant`. They are not the same, they have different properties. This is probably not _completely_ broken because they both share some property names but it's conceptually wrong.

Reply:
You are right, good catch. I've reverted this.